packages/e2e-tests/specs/block-transforms.test.js

```diff
@@ -96,28 +95,7 @@ const getTransformResult = async ( blockContent, transformName ) => {
 };
 
 describe( 'Block transforms', () => {
-	// Todo: Remove the filter as soon as all fixtures are corrected,
-	// and its direct usage on the editor does not trigger errors.
-	// Currently some fixtures trigger errors (mainly media related)
-	// because when loaded in the editor,
-	// some requests are triggered that have a 404 response.
-	const fileBasenames = filter(
-		getAvailableBlockFixturesBasenames(),
-		( basename ) => (
-			! some(
-				[
-					'core__image',
-					'core__gallery',
-					'core__video',
-					'core__file',
-					'core__media-text',
-					'core__audio',
-					'core__cover',
-				],
-				( exclude ) => basename.startsWith( exclude )
-			)
-		)
-	);
+	const fileBasenames = getAvailableBlockFixturesBasenames();
```

Comment:
How about you update the failing test (https://travis-ci.com/WordPress/gutenberg/jobs/187619473#L1011):
> Block transforms › should contain the expected transforms

to perform more relaxed check using [toMatchObject](https://jestjs.io/docs/en/expect#tomatchobjectobject):

```js
expect(
	transforms
). toMatchObject( EXPECTED_TRANSFORMS );	
```

Reply:
I tried this locally and it seems to work:

```js
it( 'example', () => {
		const actual = {
			core__gallery__columns: {
				availableTransforms: [
					'Slideshow',
					'Image',
				],
				originalBlock: 'Gallery',
			},
		};
		const expected = {
			core__gallery__columns: {
				availableTransforms: expect.arrayContaining( [
					'Image',
				] ),
				originalBlock: 'Gallery',
			},
		};

		expect( actual ).toMatchObject( expected );
	} );
```